content/docs/programs/immunizations.md

```diff
@@ -94,13 +71,35 @@ To add a new Vaccine Course, click the `Add vaccine course` button in the top ri
 
 ![vaccine course: add button](/docs/programs/images/vaccine_course_add_button.png)
 
-This will open a new window, where you can enter the name of the new Vaccine Course.
+This will open a new window, where you can enter the details of the new Vaccine Course, including its name, details like its wastage rate, and the dose schedule.
```

Comment:
```suggestion
This will open a new window, where you can enter the details of the new Vaccine Course, e.g. Vaccine course name, wastage rate, dose schedule.
```